backend/jottacloud/jottacloud.go

```diff
@@ -373,6 +373,11 @@ func v2config(ctx context.Context, name string, m configmap.Mapper) {
 	fmt.Printf("Login Token> ")
 	loginToken := config.ReadLine()
 
+	// We may have these left over from the legacy config. Token refresh will break if these are
+	// set so have to remove them from the config
+	m.Delete(configClientID)
```

Comment:
It occurs to me that if just set them to empty string then most of this commit isn't needed.

In general the config system can't distinguish between empty string and key missing, so maybe we should just set them to empty and lose the rest of the commit?